lib/KBParallel/utils/job.py

```diff
@@ -85,12 +92,13 @@ def check_local_status(self):
             status = {'finished': 1, 'job_state': 'suspend', 'error': str(err)}
         return status
 
-    def check_njs_status(self):
+    def check_ee2_status(self):
         """Check the result of a job running on NJS."""
-        response = self.njsw.check_jobs({
-            'job_ids': [self.job_id],
-            'with_job_params': 0
-        })
-        job_state = response['job_states'][self.job_id]
+
+        cjp = {'job_id': self.job_id, 'projection': []}
+        response = self.ee2.check_job(params=cjp)
+        from pprint import pprint
+        pprint(response)
+        job_state = response
```

Comment:
no reason to have this additional assignment here

Reply:
fixed